plugins/SLURM-Integration/install.sh

```diff
@@ -26,20 +26,56 @@ INSTALLATION_FAILED='Installation failed'
 install_status=0
 WORK_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
+# Get the current PATH
+export PATH=$PATH:/usr/bin:/usr/local/bin:/opt/local/bin
+
 errorlog() {
     echo "ERROR: $1"
 }
+function is_root_user()
+{
+    if [ ! "$(id -u)" = "0" ]; then
```

Comment:
```id``` is linux binary that not always installed, better to use bash/sh built-in env variables like ```EUID``` which is always be correct